test/github-issues/182/entity/Post.ts

```diff
@@ -12,7 +12,7 @@ export class Post {
     @Column()
     title: string;
 
-    @Column({ nullable: false })
+    @Column("int", { nullable: false })
```

Comment:
Why was that needed? Shouldn't change much but don't see any obvious reason.

Reply:
This test was a little tricky. It implies it's testing an enum type, but it never creates an enum column in the db, just uses an enum in the orm code. When the code is transpiled, for whatever reason the column is created as an int, but running with ts-node it was sending `object` to mysql as the column type, which of course doesn't work. Adding `int` there made the test execute correctly - the same SQL statements are sent to the db for both the transpiled and ts-node versions.

Reply:
really useful to add according comment :)